pyclesperanto_prototype/_tier2/_minimum_of_masked_pixels.py

```diff
@@ -0,0 +1,73 @@
+from .._tier0 import Image
+from .._tier0 import create
+from .._tier0 import create_2d_xy
+from .._tier0 import create_2d_yx
+from .._tier0 import execute
+from .._tier1 import transpose_xz
+from .._tier1 import transpose_yz
+from .._tier0 import pull
+from .._tier1 import copy_slice
+
+def __minimum_of_masked_pixels_reduction(input : Image, input_mask : Image, output : Image, output_mask : Image):
+    parameters = {
+        "dst_min": output,
+        "dst_mask": output_mask,
+        "mask": input_mask,
+        "src": input
+    }
+
+    execute(__file__, 'minimum_of_masked_pixels_3d_2d_x.cl', 'minimum_of_masked_pixels_3d_2d', output.shape, parameters)
+
+def _slice_to_stack(image : Image, stack : Image = None, num_slices = 1):
+    # TODO: This might become a core method
+    if stack is None:
+        stack = create([num_slices, image.shape[0], image.shape[1]])
+
+    for i in range(0, num_slices):
+        copy_slice(image, stack, i)
+    return stack
```

Comment:
Hm, now that I'm thinking about this. I think, it should just work using OpenCLs CLAMP_TO_EDGE strategy of which pixel to access in case the stack is to short in Z... I need to dig a bit deeper in numpy to see if this holds true. Thanks for the links!